docs/content/latest/explore/ysql-language-features/tablespaces.md

```diff
@@ -13,154 +13,220 @@ isTocNested: true
+## Create a single-zone table
 
-The following example demonstrates how to create a tablespace in which the value of `min_num_replicas` does not correspond to `num_replicas`:
+By default creating any tables in the cluster created above will spread all of its data across all regions. By contrast, let us create a table and constrain all of its data within a single zone using tablespaces. The placement policy that we will use can be illustrated using the following diagram:
+
+![Single Zone Table](/images/explore/tablespaces/single_zone_table.png)
+Now let us create a tablespace outlining the above placement policy and a table associated with that tablespace:
 
 ```sql
-CREATE TABLESPACE us_east_tablespace 
-WITH (replica_placement='{"num_replicas": 5, "placement_blocks":
-[{"cloud":"aws","region":"us-east","zone":"us-east-1a","min_num_replicas":1},
-{"cloud":"aws","region":"us-east","zone":"us-east-1b","min_num_replicas":1},
-{"cloud":"aws","region":"us-east","zone":"us-east-1c","min_num_replicas":1}]}');
+CREATE TABLESPACE us_east_1a_zone_tablespace
+  WITH (replica_placement='{"num_replicas": 3, "placement_blocks": [
+    {"cloud":"aws","region":"us-east-1","zone":"us-east-1a","min_num_replicas":3}]}');
+
+CREATE TABLE single_zone_table (id INTEGER, field text)
+  TABLESPACE us_east_1a_zone_tablespace SPLIT INTO 1 TABLETS;
 ```
 
-In the preceding example, the three `min_num_replicas` fields sum up to 3, whereas the total required replication factor is 5. As a result, even though `us_east_tablespace` has been created successfully, a notice is displayed after the execution stating that the additional two replicas are to be placed at the YB-Load balancer’s discretion in  `aws.us-east.us-east-1a`, `aws.us-east.us-east-1b`, and `aws.us-east.us-east-1c`. Based on the load, `aws.us-east.us-east-1a` may have three replicas and the other zones may have one, or it might later change such that `aws.us-east.us-east-1a` and `aws.us-east.us-east-1b` have two replicas each, whereas `aws.us-east.us-east-1c` has one replica. YB-Load balancer always honours the value of `min_num_replicas` and have at least one replica in each cloud.region.zone, and the additional replicas may be moved around based on the cluster load. 
+Note from the above cluster configuration that the nodes in us-east-1a were 172.152.29.181, 172.152.27.126 and 172.152.22.180. By navigating to the table view in the YB-Master UI, we can verify that the tablet created for this table was indeed placed in us_east_1a_zone:
 
-### How to Create a Single-Zone Tablespace
+![YB-Master UI: Tablets of single_zone_table](/images/explore/tablespaces/single_zone_table_tablet_distribution.png)
 
-To highlight the difference between multi-zone and single-zone tablespaces, the following example shows how to create a single-zone tablespace:
+Now let us measure the latencies incurred for INSERTs and SELECTs on this table, where the client is in us-east-1a zone:
```

Comment:
The following measures the latencies incurred for INSERTs and SELECTs on this table, where the client is in us-east-1a zone: